framework/src/modules/nft/events/create.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { BaseEvent, EventQueuer } from '../../base_event';
+import { LENGTH_COLLECTION_ID, LENGTH_NFT_ID, NftEventResult } from '../constants';
+
+export interface CreateEventData {
+	address: Buffer;
+	nftID: Buffer;
+	collectionID: Buffer;
+}
+
+export const createEventDataSchema = {
```

Comment:
```suggestion
export const createEventSchema = {
```
align with names in token module. same for other places where applicable.